tests/test_action_find_ip.py

```diff
@@ -0,0 +1,24 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+# from mock import MagicMock
+
+from aci_base_action_test_case import CiscoACIBaseActionTestCase
+
+from find_ip import FindIP
+
+
+class GetAPsActionTestCase(CiscoACIBaseActionTestCase):
```

Comment:
This is good boiler plate, but now you need some actual test functions that perform tests on your code.